src/nxdoc/nuxeo-server/installation/standard-high-availability-nuxeo-cluster-architecture.md

```diff
@@ -0,0 +1,349 @@
+    -
+        author: Solen Guitter
+        date: '2012-09-06 15:39'
+        message: ''
+        version: '11'
+    -
+        author: Solen Guitter
+        date: '2011-01-07 11:19'
+        message: added toc
+        version: '10'
+    -
+        author: Solen Guitter
+        date: '2011-01-07 11:18'
+        message: ''
+        version: '9'
+    -
+        author: Thierry Delprat
+        date: '2010-10-19 13:54'
+        message: ''
+        version: '8'
+    -
+        author: Thierry Delprat
+        date: '2010-10-19 13:49'
+        message: ''
+        version: '7'
+    -
+        author: Thierry Delprat
+        date: '2010-06-14 12:02'
+        message: ''
+        version: '6'
+    -
+        author: Thierry Delprat
+        date: '2010-06-14 02:41'
+        message: ''
+        version: '5'
+    -
+        author: Thierry Delprat
+        date: '2010-06-14 02:38'
+        message: ''
+        version: '4'
+    -
+        author: Thierry Delprat
+        date: '2010-05-25 12:12'
+        message: ''
+        version: '3'
+    -
+        author: Thierry Delprat
+        date: '2010-05-24 19:23'
+        message: ''
+        version: '2'
+    -
+        author: Admin name placeholder
+        date: '2010-03-01 00:55'
+        message: ''
+        version: '1'
+
+---
+{{! excerpt}}
+This page details standard architecture options to deploy a Nuxeo cluster.
+{{! /excerpt}}
+
+{{#> callout type='info' heading='Before Reading'}}
+This page assumes you are already familiar with the different components of a Nuxeo cluster. If not, you should have a look at our [Nuxeo architecture introduction]({{page page='nuxeo-architecture-introduction'}}) page first.
+{{/callout}}
+
+## High Availability Production Architecture
+
+![]({{file name='nuxeo-cluster-logical-architecture.png'}} ?border=true)
+
+The standard Nuxeo cluster architecture providing high availability is composed of:
+1. Two load balancers in front with sticky sessions handle incoming requests and direct them to the appropriate Nuxeo server nodes.
+1. A reverse proxy to protect each Nuxeo server node (usually Apache or Nginx).
+1. At least two Nuxeo server nodes are available. You can add any number of nodes to scale out performances.
+1. At least three nodes are available for the Elasticsearch cluster. Same for Redis. Contrarily to Nuxeo server nodes, these two components always require an odd number of nodes to avoid the split-brain problem, which means you need to add nodes by batches of two minimum when wishing to scale out performances.
+1. A database system providing high availability. Each solution has its own options for this, therefore we can't go into further details here.
+1. A shared file system that is used to store binary files.
+
+### Deploying in Cloud or Container Based Deployment
+
+This diagram translates perfectly to an on premise deployment using a container based technology like docker and we provide a ready to use docker image for Nuxeo Server.
```

Comment:
Yes:
https://doc.nuxeo.com/nxdoc/setting-up-your-nuxeo-environment/#docker
seems to be a good candidate.

We can also link to our docker repo:
https://hub.docker.com/_/nuxeo/